content/community/team/matt.md

```diff
@@ -0,0 +1,14 @@
+---
+author: true
+kind: other
+alias: mattj-io
+name: Matt
+avatar: matt.png
+about:
+  Matt is an experienced technologist focused on the open source space with wide ranging experience across technical and management roles. He's built and led teams and organisations from startups to global enterprises, successfully brought products and services to market, and created innovation and business value across a wide spectrum of different technologies.
```

Comment:
Matt runs the Open Source Program Office at D2iQ. He's a regular speaker at conferences and meetups all over the world, and has been building stuff with open source software for longer than he cares to remember.